components/dash-table/tests/visual/percy-storybook/DashTable.percy.tsx

```diff
@@ -165,10 +179,21 @@ storiesOf('DashTable/Fixed Rows & Columns', module)
             columns={columnsA2J}
             fixed_rows={{headers: true}}
             row_deletable={true}
-            row_selectable={true}
+            row_selectable='single'
             style_data_conditional={style_data_conditional}
         />
     ))
+    .add('with 1 fixed row, longer header width than content cell', () => (
+        <DataTable
+            setProps={setProps}
+            id='table'
+            data={dataWithLongerTitle}
+            columns={columnsWithLongerTitle}
+            fixed_rows={{headers: true}}
+            row_deletable={true}
+            row_selectable='single'
+        />
```

Comment:
![image](https://user-images.githubusercontent.com/1280389/134088712-4b8b61d9-6272-4d22-be2b-c1a58c5fd93b.png)

Could we add a test where the columns are as wide as `max(header, cell)`? In this test case, the table is very wide so we don't see the issue of the narrow headers.

Reply:
Perhaps a test with several columns

Reply:
Ah well I guess this existing test covers it:
![image](https://user-images.githubusercontent.com/1280389/134089279-23d699c4-520d-4b01-9009-e9b50fac3e43.png)